src/log.py

```diff
@@ -0,0 +1,63 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import os.path
+import time
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from vehicle.
+    """
+    def __init__(self, vehicle_name):
+        self.file_name = ""
+        self.class_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+        if vehicle_name.lower() == 'follower':
+            self.class_name = vehicle_name
+        elif vehicle_name.lower() == 'leader':
+            self.class_name = vehicle_name
+        else:
+            raise ValueError('Unexpected class name')
+
+        self.file_name = self.class_name + self._start_time + '.log'
+
+
+    def write_to_file(self, dictionary):
+        """
+        Write vehicle data into text file.
```

Comment:
This can do more than just write vehicle data to a text file. It can write any dictionary to a text file.

Reply:
Changed comment to any dictionary